### DISCUSSION TITLE
请问如何手动触发EditableRow/EditableCell的form.validateFields校验

### DISCUSSION THREAD

**Discussion by quicklearning123** [ant-design/ant-design#30852] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/table/#components-table-demo-edit-cell](https://ant.design/components/table/#components-table-demo-edit-cell)

### Steps to reproduce
必现。
组件初始化时填充了一批数据，想对其校验合法性。

### What is expected?
能主动触发form校验

### What is actually happening?
1.不知道在EditableRow外部怎么主动触发
2.只有点击单元格切成编辑态才能触发校验

![image](https://user-images.githubusercontent.com/70048681/120641663-7155dd80-c4a6-11eb-8be9-46282a8baa5e.png)

| Environment | Info |
|---|---|
| antd | 4.16.1 |
| React | 16 |
| System | mac |
| Browser | chrome 88 |

**Comment by yoyo837** [comment]
其实是思路问题：首先默认demo里非编辑状态是没有渲染Form.Item的，也就意味着你不能通过Form这条路来触发验证；所以最简单的两条路，1是表格里始终渲染成编辑状态，2自己实现一下非编辑状态的数据验证并能提示到对应格里；

最后，antd的组件个人认为未来不太会直接支持你想要的这个效果，因为没有那样设计交互；

**Reply by peacejj** [reply]
我采用了第一种方案，始终是编辑状态，但依然不清楚怎么在外层调用

**Reply by peacejj** [reply]
因为焦点只可能在一个输入框，多个输入框的情况下，怎么触发全部校验呢

**Comment by yoyo837** [comment]
我自己项目就有这种场景
